src/components/menu/Menu.tsx

```diff
@@ -51,6 +42,8 @@ export interface MenuProps<S = any> extends Omit<BoxProps, 'children' | 'classNa
   selected?: S;
 }
 
+type Position = Exclude<MenuProps['position'], undefined>;
```

Comment:
This does not look right. (Could just be `type Position = MenuProps['position']` I guess)

PP:
```
type Position = 'auto' | 'static' | 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';

export interface MenuProps<S = any> extends Omit<BoxProps, 'children' | 'className'> {
  ...
  position?: Position
  ...
}
```

Reply:
@LennertBlommaert the type is correct, but your suggestion makes more sense.